tests/test_registrations/test_archiver.py

```diff
@@ -1130,7 +1130,7 @@ def test_archiving_nodes_not_added_to_search_on_archive_failure(self, mock_send,
                 mock.patch('website.archiver.model.ArchiveJob.success', mock.PropertyMock(return_value=False))
         ) as (mock_finished, mock_sent, mock_success):
             listeners.archive_callback(reg)
-        mock_update_search.assert_not_called()
+        mock_delete_index_node.assert_called()
```

Comment:
You obviously don't need to fix all of these, but a few are sneaking in in this PR. Please don't let them.